changelogs format fixes (#107)
agno-agi/agno-docs: changelog/overview.mdx

## Bug Fixes:

- **Agent Dataclass Compatibility:** Removed `slots=True` from the agent dataclass decorator, which was not compatible with Python < 3.10.

> - **AzureOpenAIEmbedder:** Fixed issue where `AzureOpenAIEmbedder` was not correctly made a dataclass.

Reviewer: ```suggestion
- **AzureOpenAIEmbedder:** Made `AzureOpenAIEmbedder` a dataclass to match other embedders.
```
